.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -18,3 +18,14 @@ A clear and concise description of any alternative solutions or features you've
 
 **Additional context**
 Add any other context or screenshots about the feature request here.
+
+---
+
+**DoD checklist**
```

Comment:
Maybe we can add license update here also for new features? Also we can check if all conversations in PR were resolved.

Reply:
Isn't it related to `COMPONENTS.md updated / doesn't need to be updated` point?